scripts/page_browse.js

```diff
@@ -420,7 +420,7 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false) {
                     }
 
                     const imageDiv = $("<div>");
-                    imageWidget = makeImageWidget(imageDiv);
+                    imageWidget = makeImageWidget(imageDiv, "C");
```

Comment:
isn't "C" the default?

Reply:
Yes, at an intermediate stage I added another parameter so needed to specify this one, then removed it again and later sqaushed all the commits, but it got left behind.